data/src/main/java/io/homeassistant/companion/android/data/integration/IntegrationRepositoryImpl.kt

```diff
@@ -102,6 +103,11 @@ class IntegrationRepositoryImpl @Inject constructor(
         return localStorage.getString(PREF_WEBHOOK_ID) != null
     }
 
+    override suspend fun getUiUrl(isInternal: Boolean): URL? {
+        return localStorage.getString(PREF_REMOTE_UI_URL)?.toHttpUrl()?.toUrl()
```

Comment:
We can use `isInternal` here already, by making sure we only return `authenticationRepository.getUrl()` if it's true.

Reply:
Done.